.travis.yml

```diff
@@ -3,7 +3,16 @@ services: docker
 language: ruby
 rvm:
   - 2.5.1
-cache: bundler
+cache:
+  bundler: true
+  directories:
+    - node_modules
+
+env:
+  global:
+    - PATH="$PATH:$TRAVIS_BUILD_DIR/node_modules/.bin"
```

Comment:
This as well

Reply:
Travis couldn't locate the path of node modules, so there was need to specify it.

Reply:
No, use `npm install -g csslint` then, it should work fine.